src/components/Footer.tsx

```
@@ -1,77 +1,110 @@
-            <h3 className="text-lg font-semibold text-gray-900 dark:text-white mb-4">
-              Henry Ssekibo
-            </h3>
-            <p className="text-gray-600 dark:text-gray-300">
-              IT Support & Digital Transformation Specialist
+    <footer className="bg-white dark:bg-zinc-950 border-t border-zinc-200 dark:border-zinc-800">
+      <div className="container-custom">
+        {/* Main Footer */}
+        <div className="py-16 grid md:grid-cols-12 gap-12">
+          {/* Brand */}
+          <div className="md:col-span-5">
+            <a href="#" className="text-2xl font-bold text-zinc-900 dark:text-white tracking-tight">
```

Biome (2.1.2)

[error] Provide a valid value for the attribute href.

The href attribute should be a valid a URL
Check this thorough explanation to better understand the context.

(lint/a11y/useValidAnchor)